lib/generateClass.js

```diff
@@ -301,29 +301,42 @@ function writeToFhir(def, fhir, cw) {
 
    /* UNCOMMENT BELOW CODE TO JAM IN DUMMY MAPPING DATA */ 
 
-    // if(fhirProfile.type === 'Patient'){
-    //   const addMapping = (path, map) => { fhirProfile.snapshot.element.find(f => f.path === path).mapping.push(map)};
-    //   addMapping('Patient.active', {'identity': 'shr', 'map': '<shr.fhir.ActiveFlag>'});
-    //   addMapping('Patient.birthDate', {'identity': 'shr', 'map': '<Value>'});
+    // const addMapping = (path, map) => {
+    //   const el = fhirProfile.snapshot.element.find(f => f.path === path);
+    //   el.mapping = el.mapping.filter(m => m.identity !== 'shr');
+    //   el.mapping.push(map);
+    // };
+    // if(fhirProfile.name === 'PatientEntryProfile'){
+    //   // addMapping('Patient.active', {'identity': 'shr', 'map': '<shr.simple.BooleanValue>'}); // DONE
+    //   // addMapping('Patient.birthDate', {'identity': 'shr', 'map': '<Value>'}); // DONE
+    //   addMapping('Patient.name.text', {'identity': 'shr', 'map': '<shr.simple.StringValue>'});
+    // }
+    // if(fhirProfile.name === 'PractitionerEntryProfile'){
+    //   addMapping('Practitioner.active', {'identity': 'shr', 'map': '<shr.simple.DoubleNestedBooleanValue>.<shr.simple.NestedBooleanValue>.<shr.simple.BooleanValue>'});
+    //   addMapping('Practitioner.name.text', {'identity': 'shr', 'map': '<shr.simple.NestedStringValue>.<shr.simple.StringValue>'});
     // }
 
     for (element of fhirProfile.snapshot.element) {
       const mapping = element.mapping && element.mapping.find(m => m['identity'] === 'shr');
 
       if(mapping !== undefined){
-        const match = mapping.map.match(/\<([^\>]*)>/);
+        // const match = mapping.map.match(/\<([^\>]*)>/);
+        const match = mapping.map.match(/\<([^\>]*)>/g)
```

Comment:
Why are two of the lesser-than/greater-than characters escaped, while the last one isn't? I'm not sure any of them need to be, based on some quick testing.